microsoft-edge/visual-studio-code/debugger-for-edge.md

```diff
@@ -1,17 +1,25 @@
 ---
-description: How to debug Microsoft Edge from Visual Studio Code
-title: Debug Microsoft Edge from Visual Studio Code
+description: You can debug Microsoft Edge from Visual Studio Code using the Debugger Extension
```

Comment:
@captainbrosset - was your plan to remove the focus from the Debugger Extension and focus on the built-in capabilities of VS Code instead?

Reply:
There is no such thing as the debugger extension anymore. So I think this description needs to be updated.
VS Code can debug JS code in Edge/Chrome/WebView/Node by default now. No need for an extension anymore, this is built-in.

Reply:
done